docs/source/migration-guides.rst

```diff
@@ -0,0 +1,258 @@
+Migration guides
+================
+
+Migration from Waffle 2.2.0 to Waffle 2.3.0
+-------------------------------------------
+
+**Created monorepo**
+
+Packages :code:`waffle-cli`, :code:`waffle-chai`, :code:`waffle-provider`, :code:`waffle-compiler` were migrated to monorepo.
+
+**Created MockProvider class**
+
+We added MockProvider class, it was change the way to create the provider.
+
+*Waffle 2.2.0*
+
+.. code-block:: ts
+
+  await createMockProvider(options);
+
+*Waffle 2.3.0*
+
+.. code-block:: ts
+
+  provider = new MockProvider();
+
+
+:code:`new MockProvider` returned :code:`providers.Web3Provider`, which is the parent class of :code:`MockProvider`.
```

Comment:
How is this useful for the reader? What do they have to do? What are the consequences?